src/lib/viewers/media/DashControls.tsx

```diff
@@ -4,14 +4,17 @@ import HDBadge from '../controls/media/HDBadge';
 import MediaFullscreenToggle, { Props as MediaFullscreenToggleProps } from '../controls/media/MediaFullscreenToggle';
 import MediaSettings, { Props as MediaSettingsProps } from '../controls/media/MediaSettings';
 import PlayPauseToggle, { Props as PlayControlsProps } from '../controls/media/PlayPauseToggle';
+import SubtitlesToggle, { Props as SubtitlesToggleProps } from '../controls/media/SubtitlesToggle';
 import TimeControls, { Props as TimeControlsProps } from '../controls/media/TimeControls';
 import VolumeControls, { Props as VolumeControlsProps } from '../controls/media/VolumeControls';
+import { SUBTITLES_OFF } from '../controls/media/MediaSettingsMenuSubtitles';
```

Comment:
Can we avoid importing this in the top-level DashControls? Can this logic live in the SubtitlesToggle, instead?